sdks/python/apache_beam/transforms/stats.py

```diff
@@ -234,3 +239,419 @@ def extract_output(accumulator):
 
   def display_data(self):
     return {'sample_size': self._sample_size}
+
+
+class ApproximateQuantiles(object):
+  """
+  PTransfrom for getting the idea of data distribution using approximate N-tile
+  (e.g. quartiles, percentiles etc.) either globally or per-key.
+  """
+
+  @staticmethod
+  def _display_data(num_quantiles, compare, key, reverse):
+    return {
+        'num_quantiles': DisplayDataItem(num_quantiles, label="Quantile Count"),
+        'compare': DisplayDataItem(compare.__class__,
+                                   label='Record Comparer FN'),
+        'key': DisplayDataItem(key.__class__, label='Record Comparer Key'),
+        'reverse': DisplayDataItem(reverse.__class__, label='Is reversed')
```

Comment:
reverse is a boolean. We would like to capture whether it is True or False, instead of it being a type bool.

Similarly for compare, __class__ will be of type None or function and that is less helpful. (For that data could be compare.__name__ for example if it is not None.)

Similarly for key,  we should try to have some meaningful thing to display to users.

Reply:
Done!